chart/templates/deployment.yaml

```diff
@@ -39,12 +42,19 @@ spec:
           httpGet:
             path: /ping
             port: {{ .Values.service.http.internalPort }}
-        volumeMounts:
-        - mountPath: /var/run/docker.sock
-          name: docker-socket
+      - name: dind-daemon 
+        image: docker:dind 
+        resources: 
```

Comment:
we can just remove this for now. This **may** be why you were seeing performance hits with this PR. I'll give this a spin with and without the resource constraints. :)